.github/workflows/clang-format.yml

```diff
@@ -9,17 +9,27 @@ jobs:
       matrix:
         path:
           - check: '.'
-            exclude: './.*/.*trace.h|./NetKVM|./viocrypt|./viofs|./viosock|./VirtIO'
+            exclude: >-
+              ./.*/.*trace.h|
+              ./NetKVM/Common/trace.h|
+              ./NetKVM/Common/virtio_net.h|
+              ./NetKVM/linux/if_ether.h|
+              ./viocrypt/sys/Wpptrace.h|
+              ./viofs/shared/fuse.h|
+              ./viofs/svc/scope_exit.h|
+              ./viosock/viosock-wsk-test/wpp-trace.h|
+              ./viosock/wsk/wpp-trace.h|
+              ./viostor/virtio_stor_trace.h
```

Comment:
```
./NetKVM/Common/trace.h
./viocrypt/sys/Wpptrace.h
./viosock/viosock-wsk-test/wpp-trace.h
./viosock/wsk/wpp-trace.h
./viostor/virtio_stor_trace.h
```

are redundant, if not fix the regexp `./.*/.*trace.h`

Reply:
./viostor/virtio_stor_trace.h is definitely  redundant, because viostor CI was enabled before and pass